test/plots.jl

```diff
@@ -11,3 +11,19 @@ bitstring_hist(r; nlargest = 10)
 space = Subspace(5, randperm(1 << 5)[1:10] .- 1)
 r = rand_state(space)
 bitstring_hist(r; nlargest = 10)
+
+@testset "plot_densities" begin
```

Comment:
can we also add a test that doesn't throw an error to make sure we cover that branch?